src/modules/auth/services/auth.service.ts

```
@@ -0,0 +1,53 @@
+
+    // Use the HashingService to compare the password
+    const isValid = await this.hashingService.comparePasswords(password, user.password);
+    if (!isValid) {
+      throw new UnauthorizedException('Invalid credentials');
+    }
+
+    return user;
+  }
+
+  async findById(id: number): Promise<User | undefined> {
+    return this.users.find(user => user.id === id);
```

ESLint

[error] Replace `user` with `(user)`

(prettier/prettier)